### PR TITLE
code refactoring. Added some kind of documentation

### PR DESCRIPTION
**Pull request by monatanasov**

### CODE REVIEW

**Comment by monatanasov** [comment]
Nothing in common as on Saturday

File: binarymode.php [hunk 1]
```diff
@@ -1,7 +1,23 @@
 <?php
 include './dbconn.php';
-$quotequery=mysqli_query($conn, "SELECT *, authors.id AS aut_id FROM `authors` LEFT JOIN `quotes` ON authors.id=quotes.author_id ORDER BY RAND() LIMIT 1 ");
-$authorquery=mysqli_query($conn, "SELECT *, authors.id AS aut_id FROM `authors` LEFT JOIN `quotes` ON authors.id=quotes.author_id ORDER BY RAND() LIMIT 1 ");
+//store all db tables in a variable
```
Comment (didotsonev) [inline binarymode.php]:
```suggestion
// store all db queries in a variable
```

File: binarymode.php [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
+$quotesQuery=mysqli_query($conn, "SELECT * FROM `quotes` ");
```
Comment (didotsonev) [inline binarymode.php]:
```suggestion
$quotesQuery=mysqli_query($conn, "SELECT * FROM `quotes`");
```

File: binarymode.php [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
+$authorsQuery=mysqli_query($conn, "SELECT * FROM `authors` ");
```
Comment (didotsonev) [inline binarymode.php]:
```suggestion
$authorsQuery=mysqli_query($conn, "SELECT * FROM `authors`");
```

File: binarymode.php [hunk 4]
```diff
@@ -1,7 +1,23 @@
...
+
+//loop each row of Quotes querry variable and store its data into multidimensional array
```
Comment (didotsonev) [inline binarymode.php]:
```suggestion
//loop each row of Quotes query variable and store its data into multidimensional array
```

File: binarymode.php [hunk 5]
```diff
@@ -1,7 +1,23 @@
...
+while($row=mysqli_fetch_assoc($quotesQuery)){
+    $quotesArray[]=$row;
+}
+//get the random quote position from the array
+$posQuery=rand(0,sizeof($quotesArray)-1);
+
+//loop each row of Authors querry variable and store its data into multidimensional array
```
Comment (didotsonev) [inline binarymode.php]:
```suggestion
//loop each row of Authors query variable and store its data into multidimensional array
```

File: binarymode.php [hunk 6]
```diff
@@ -25,34 +41,52 @@
...
+    <script>
+        //create 2 js variables to store php results for additional comparison
+        let quotesResultID = <?php echo(json_encode($quotesArray[$posQuery]['author_id'])); ?>;
+        let authorsResultID = <?php echo(json_encode($authorsArray[$posAuthor]['id'])); ?>;
+        //a variable which extracts Author-Name and show it in the alert box after
+        let authorResultName = <?php echo(json_encode($authorsArray[$posAuthor]['name'])); ?>;
+        //function for comparison the YES result
+        function myFunction() {
+            if(quotesResultID===authorsResultID){
+
```
Comment (didotsonev) [inline binarymode.php]:
```suggestion
```

**Pull request merged**